application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -46,16 +46,27 @@ Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interva
 Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
 
-# Live table keys
+## Most Viewed Pages
+Analytics.Code.Macros.label=Page Title
+Analytics.Code.Macros.nb_visits=Unique Vistators
+Analytics.Code.Macros.nb_hits=Page Views
+Analytics.Code.Macros.sum_time_spent=Time spent(s)
+Analytics.Code.Macros.entry_nb_visits=No. of views that started on this page
+Analytics.Code.Macros.entry_nb_actions= No. Of Actions
+Analytics.Code.Macros.bounce_rate=Bounce Rate
+Analytics.Code.Macros.exit_rate=Exit Rate
```

Comment:
You still need to work a little on the translations keys. There was also a link to https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/ inside the other one. Key format is the important part. Look at other extensions for examples
Also, for the translations you don't use the same case (upper or lower) for all the words, it's not consistent